tests/negative/Termination/NoLexOrder.mjuxix

```diff
@@ -0,0 +1,11 @@
+module TerminatingKeyword;
```

Comment:
This fails scoping because the filename and the module name do not match (and the extension is `mjuxix` instead of `mjuvix`, not because of termination.

Reply:
Actually - I'm not sure even if I change the module name. How do I see this fails termination?

Also, are these tests run by `stack test`?

Reply:
The test suite doesn't include termination. I know. This issue will be fixed in a future release, which needs to update the pipeline for running the termination checker just after `upToAbstract`.

Reply:
You can see that if the termination fails by running `minijuvix termination graph NameFile.mjuvix`.